clang/docs/UsersManual.rst

```diff
@@ -5272,6 +5296,8 @@ The Visual C++ Toolset has a slightly more elaborate mechanism for detection.
 Restrictions and Limitations compared to Clang
 ----------------------------------------------
 
+.. _clang_cl_strict_aliasing:
+
 Strict Aliasing
```

Comment:
```suggestion
Strict Aliasing in ``clang-cl``
```

But consider just merging this into the new section above.

Reply:
I shortened the section here, linking back to the main section. I kept a sentence here to call out the difference. WDYT?